crates/bevy_hierarchy/src/lib.rs

```diff
@@ -23,11 +23,19 @@ pub mod prelude {
 }
 
 use bevy_app::prelude::*;
+use bevy_ecs::schedule::SystemLabel;
 
 /// The base plugin for handling [`Parent`] and [`Children`] components
 #[derive(Default)]
 pub struct HierarchyPlugin;
 
+/// Label enum for the systems relating to hierarchy upkeep
+#[derive(Debug, Hash, PartialEq, Eq, Clone, SystemLabel)]
+pub enum HierarchySystem {
```

Comment:
Why did this get included in this PR? Does CI break without it?

Reply:
Good question, that must have shown up while I was rebasing. Not sure why.